backend-base/README.md

```diff
@@ -31,25 +35,6 @@ mvn test
 
 ###  Native-mode integration testing
 
-#### Prerequisites
```

Comment:
At the moment, the [native integration tests](https://github.com/che-incubator/che-workspace-telemetry-client/blob/master/backend-base/src/test/java/org/eclipse/che/incubator/workspace/telemetry/NativeTelemetryResourceIT.java) don't require a running devworkspace to pass, since requests return 200 even if the code fails to find the DevWorkspace in a cluster.